### PR TITLE
NEP: NeoContract Manifest

### PR DESCRIPTION
**Pull request by bettybao1209**
close https://github.com/neo-project/proposals/issues/110

### CODE REVIEW

**Comment by shargon** [comment]
Thanks for my inclusion @bettybao1209 ! I think that the document describe well the manifest format. Good work!

**Comment by igormcoelho** [comment]
I'm honored for the inclusion, thanks!

File: nep-14.mediawiki [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<pre>
+  NEP: 14
+  Title: NeoContract Manifest
+  Author: Erik Zhang <erik@neo.org>, shargon <shargon@gmail.com>, Vitor Nazário Coelho <vncoelho@gmail.com>, Igor Machado Coelho <igormachado@gmail.com>, Li Jianying <lightsever@hotmail.com>
```
Comment (erikzhang) [inline nep-14.mediawiki]: Should this be changed to your real name? @shargon

Comment (shargon) [inline nep-14.mediawiki]:
```suggestion
  Author: Erik Zhang <erik@neo.org>, Fernando Díaz Toledano <shargon@gmail.com>, Vitor Nazário Coelho <vncoelho@gmail.com>, Igor Machado Coelho <igormachado@gmail.com>, Li Jianying <lightsever@hotmail.com> 
```

**Comment by erikzhang** [comment]
Merge?

**Comment by erikzhang** [comment]
It should be merged after #119.

**Comment by shargon** [comment]
How will now the user that NEP3 and NEP6 it's obsolete? we should add a line for redirect to the new one

**Review by shargon — DISMISSED** [review]

**Comment by luc10921** [comment]
[#2060](https://github.com/neo-project/neo/pull/2060) removed `features` and [#2024 ](https://github.com/neo-project/neo/pull/2024/files) added `name`, but NEP-15 did not change it Specification to address those changes. Also, [#2044](https://github.com/neo-project/neo/pull/2044) removed `hash` from [ABI](https://github.com/neo-project/proposals/pull/119).

**Comment by gsmachado** [comment]
I was discussing With devhawk in [this issue](https://github.com/ngdenterprise/neo3-visual-tracker/issues/34) the need of specifying the file name suffix `.manifest.json` as a standard, to our community.

What I observe is that compilers give the option to developers to specify any kind of file name they want (which is not bad!), but then, at least the default output filename name should be set with a standardized suffix (i.e., `.manifest.json`).

This gains even more importance to developer experience if we consider that key tools in our community, like https://github.com/ngdenterprise/neo3-visual-tracker, relies on discovering contracts to be deployed based on the manifest filename.

I suggest adding, somewhere in this NEP:

> The default filename suffix that compilers and tools should output is `.manifest.json`.

or something in this direction.

**Comment by erikzhang** [comment]
Does anyone have any comments or suggestions?

**Review by shargon — DISMISSED** [review]

**Review by shargon — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**